components/match2/commons/match_group_display.lua

```diff
@@ -102,6 +102,34 @@ function MatchGroupDisplay.MatchGroupById(args)
 	})
 end
 
+--[[
+Displays a singleMatch specified by a bracket ID and matchID.
+]]
+function MatchGroupDisplay.MatchByMatchId(args)
+	local bracketId = args.id or args[1]
+	args.id = bracketId
+	args[1] = bracketId
+	assert(bracketId, 'Missing bracket ID')
+	assert(args.matchid, 'Missing match ID')
```

Comment:
```suggestion
	assert(args.matchid, 'Missing match ID')
	local matchIdLength = string.len(args.matchid)
	args.matchid = string.rep('0', 4 - matchIdLength) .. args.matchid
```
this would allow to input `|matchid=1` instead of `|matchid=0001` etc.